app/models/shipit/deploy_spec/npm_discovery.rb

```diff
@@ -0,0 +1,49 @@
+module Shipit
+  class DeploySpec
+    module NpmDiscovery
+      def discover_dependencies_steps
+        discover_package_json || super
+      end
+
+      def discover_package_json
+        npm_install if npm?
+      end
+
+      def npm_install
+        [%(npm install --no-progress)]
+      end
+
+      def discover_review_checklist
+        discover_npm_checklist || super
+      end
+
+      def discover_npm_checklist
+        [%(<strong>Don't forget version and tag before deploying!</strong> You can do this with:<br/>
+          npm version <strong>&lt;patch type&gt;</strong> && git push --tags</pre>)] if npm?
```

Comment:
Agreed. I left it vague because there are many more options, but anyone looking for those likely has the confidence to ignore these suggestions.